backend/app/notify/email.go

```diff
@@ -19,11 +19,11 @@ import (
 
 // EmailParams contain settings for email notifications
 type EmailParams struct {
-	From                 string // from email address
-	MsgTemplate          string // request message template
-	VerificationSubject  string // verification message subject
-	VerificationTemplate string // verification message template
-	UnsubscribeURL       string // full unsubscribe handler URL
+	From                     string // from email address
+	MsgTemplatePath          string // request message template
```

Comment:
I think, a comment should be adjusted to something like "path to request message template".